content/master/concepts/composition.md

```diff
@@ -1250,7 +1249,8 @@ so:
 [issue-2024]: https://github.com/crossplane/crossplane/issues/2024
 [xrs-and-mrs]: /media/composition-xrs-and-mrs.svg
 [how-it-works]: /media/composition-how-it-works.svg
-[crd-docs]: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/
 [provider-kubernetes]: https://marketplace.upbound.io/providers/crossplane-contrib/provider-kubernetes
 [provider-helm]: https://marketplace.upbound.io/providers/crossplane-contrib/provider-helm/
 [claims-and-xrs]: /media/composition-claims-and-xrs.svg
+[xr-ref]: {{<ref "../getting-started/introduction" >}}
```

Comment:
i feel like this isn't the right target for what this link intends.  It takes the reader to the generic introduction page, but it intends to take you to the details composition reference.

The tricky part is that I would argue we're already on the composition reference page 🤪 - all the detailed info and config options etc. are on this very page.

Perhaps the best link here would instead be to `/concepts/composition/#compositions`, which is the specific section that goes through every composition field, type, config option?  What do you think?

Reply:
and if we do then link to a section that is on this same page, we might want to change the language from "reference page" to something more like "reference section"

Reply:
I've updated the link to the suggested sub section.